tsdown.config.ts

```diff
@@ -0,0 +1,53 @@
+import { buildTemplates } from '@sveltejs/create/build';
+import MagicString from 'magic-string';
+import fs from 'node:fs';
+import path from 'node:path';
+import process from 'node:process';
+import { defineConfig } from 'tsdown';
+
+export default defineConfig({
+	cwd: 'packages/cli',
+	entry: ['lib/index.ts', 'lib/testing.ts', 'bin.ts'],
+	sourcemap: !process.env.CI,
+	dts: { oxc: true },
```

Comment:
the types from `package-manager-detector` wasn't being bundled when looking at the generated `dist/lib/testing.d.ts` file

this should fix it:
```suggestion
const pkgJson = fs.readFileSync(path.resolve('packages', 'cli', 'package.json'), 'utf8');
const pkg = JSON.parse(pkgJson);

export default defineConfig({
	cwd: 'packages/cli',
	entry: ['lib/index.ts', 'lib/testing.ts', 'bin.ts'],
	sourcemap: !process.env.CI,
	dts: {
		oxc: true,
		// setting `resolve: true` seems to anger Rolldown due to our `postcss` dep
		resolve: Object.keys(pkg.devDependencies)
	},
```